guides.md

```diff
@@ -28,6 +29,165 @@ our values.
+    - Username: `[info provided by new user]`
+    - Domain: `hypha.coop`
+    - Full name: `[info provided by new user]`
+    - Quota: `4096`
+    - Password: `[auto-generated]`
+    - Active: `checked` :ballot_box_with_check:
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of Mailcow mailbox creation popup" src="images/email-mailbox-3.png" />
+    </details>
+
+We'll now log in to confirm access, and set up an initial forwarder to make first-time usage simpler.
+
+5. Open an incognito browser and access webmail login: https://mailninja.aseriesoftubez.com/SOGo/
+6. Log into the account you just created.
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of webmail login screen" src="images/email-mailbox-4.png" />
+    </details>
+7. Navigate through these menus:  
+    _:gear:(top-left) > Mail menu item > Forward tab_
+8. Check "Forward incoming mail" and enter provided primary email, clicking "Keep a copy".
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of webmail forwarding settings" src="images/email-mailbox-5.png" />
+    </details>
+9. From your own `@hypha.coop` email, send a welcome email, like this template:
+    >
+    > Hi XXXX --
+    >
+    > Yay! Your Hypha email's set up and forwarding to your personal inbox. People can start emailing you at this address immediately, without any extra effort on your part.
+    >
+    > Your new email (and login): XXX@hypha.coop  
+    > Your password: Sent via Signal to XXX-XXX-XXXX
+    >
+    > Having said that, when you'd like to start _sending_ email as XXX@hypha.coop, you'll need to take some more steps.
+    >
+    > The next steps will depend on how you like to handle email:
+    > - [If you'd like to use it from your existing Gmail account...](#using-gmail)
+    > - [If you'd like to use it from our hosted webmail interface...](#using-the-web-interface)
+    > - [If you'd like to use it from another email client...](l#using-your-existing-email-client)
+    >
+    > Once you're set up, let us know by sending a test email to infra@hypha.coop, ideally with a short emoji-only story. (Keypad mashes a-ok!) We'll send a friendly poke in a week if we haven't heard back.
+    >
+    > Questions? Feedback? Reach out via infra@hypha.coop or [`#hyphacoop-infra:tomesh.net`](https://chat.tomesh.net/#/room/#hyphacoop-infra:tomesh.net) for chat.
+    >
+    > In Solidarity,  
+    > xxxxx
+10. Send a low-entropy Signal message with the previously noted password, for example: `🍄 password: xxxxxxxx`
+
+### Using your new inbox (Users)
+
+Accessing your inbox can be done via the hosted webmail interface or by using the same email client you're already using.
+
+What you'll need:
+- Your new email address
+- Your password (sent to you by admin on initial setup)
+
+#### Using GMail
+
+1. Log into GMail
+2. Go to GMail settings, by navigating:  
+    _:gear: (top-right) > Settings > Accounts and Import_
```

Comment:
Maybe?

```suggestion
    :gear: `(top-right) > Settings > Accounts and Import`
```

Reply:
```suggestion
    `[gear] (top-right) > Settings > Accounts and Import`
```

Guess if we try backticks here, emoji doesn't make sense to use (only kinda works here because it's first)

Reply:
emojis work if you use the emoji rather than the shortcode

Reply:
Updated!